packages/sdk/identity/src/odis/identifier.ts

```diff
@@ -0,0 +1,199 @@
+import {
+  CombinerEndpoint,
+  KEY_VERSION_HEADER,
+  SignMessageRequest,
+  SignMessageResponseSchema,
+} from '@celo/phone-number-privacy-common'
+import { soliditySha3 } from '@celo/utils/lib/solidity'
+import { createHash } from 'crypto'
+import debugFactory from 'debug'
+import { BlsBlindingClient, WasmBlsBlindingClient } from './bls-blinding-client'
+import {
+  AuthenticationMethod,
+  AuthSigner,
+  EncryptionKeySigner,
+  getOdisPnpRequestAuth,
+  queryOdis,
+  ServiceContext,
+} from './query'
+
+const debug = debugFactory('kit:odis:identifier')
+const sha3 = (v: string) => soliditySha3({ type: 'string', value: v })
+
+const PEPPER_CHAR_LENGTH = 13
+const PEPPER_SEPARATOR = '__'
+
+export enum IdentifierType {
+  PHONE_NUMBER = 0,
+  EMAIL = 1,
+  TWITTER = 2,
+  // feel free to put up a PR to add more types!
+}
+
+export function getIdentifierPrefix(type: IdentifierType) {
+  switch (type) {
+    case IdentifierType.PHONE_NUMBER:
+      return 'tel'
+    case IdentifierType.EMAIL:
+      return 'mailto'
```

Comment:
It would be good to include a link to the resources used to determine that these are appropriate prefixes